chef_master/source/upgrade_server_ha_v2.rst

```diff
@@ -30,18 +31,18 @@ Chef Backend 1.x to 2.x Upgrade
     backend-2  192.168.33.216  008782c59d3628b6bb7f43556ac0c66c  follower  follower  not_master
     backend-3  192.168.33.217  1af654172b1830927a571d9a5ba7965b  follower  follower  master
 
-   In this example, ``backend-1`` is the **leader** node, as indicated by its role in the **Role** column. 
+   In this example, ``backend-1`` is the **leader** node, as indicated by its role in the **Role** column.
 
 #. Install the new Chef Backend package on all nodes in the cluster:
 
-   * RHEL and CentOS: 
-   
+   RHEL and CentOS:
+
      .. code-block:: bash
 
         yum install PATH_TO_FILE.rpm
 
-   * Debian and Ubuntu:  
-     
+   Debian and Ubuntu:
+
      .. code-block:: bash
 
```

Comment:
Github won't let me comment directly on this section as part of a review, but in `Step 8` below the instructions have an incorrect link: 

`https://docs.chef.io/install_server_ha.html#upgrading-chef-server-on-the-frontend-machines` 
should be
`https://docs.chef.io/install_server_ha.html#upgrading-chef-infra-server-on-the-frontend-machines`

Reply:
done